### DISCUSSION TITLE
Thank you so much for building this!!!

### DISCUSSION THREAD

**Discussion by Kadrian** [kdheepak/taskwarrior-tui#481] [Show and tell]
Hey, I just wanted to let you know that I'm using taskwarrior-tui every other day and it's only with this tui that taskwarrior feels productive for me. I'm really enjoying how well the list works with VIM key bindings.

I wish I could just drag and drop the split window size between the list and the details, but that's just minor.

Just wanted to say thank you and please keep up the good work!

**Comment by kdheepak** [comment]
Thank you for the kind words!

Feel free to open that feature request as an issue for enhancement! It should be mostly straightforward to add.
